### DISCUSSION TITLE
storybook/@angular : issues with standalone component

### DISCUSSION THREAD

**Discussion by jy95** [storybookjs/storybook#24334] [Help]
Hello,

I was testing Storybook with the [library](https://github.com/jy95/ngx-json-schema-viewer) I am working on, but I encountered an issue when writting a [story](https://github.com/jy95/ngx-json-schema-viewer/blob/master/projects/ngx-json-schema-viewer/src/stories/Viewer.stories.ts) :  

![image](https://github.com/storybookjs/storybook/assets/9306961/07935a3d-7ff8-490a-854f-f7ac6ea81337)

ERROR Error: NG0201: No provider for MatExpansionPanel found in NodeInjector.

It seems weird as I set up the imports in the standalone component
```tsx

import { MatExpansionModule } from '@angular/material/expansion';

@Component({
  selector: 'ngx-json-schema-viewer',
  standalone: true,
  imports: [
    CommonModule,
    MatExpansionModule,
    CreateNodesComponent,
    ErrorOccurredLabelComponent,
    LoadingLabelComponent,
  ],
```

I'm open for ideas. 

Thanks in advance

**Comment by valentinpalkovic** [comment]
Hey @jy95 

Thank you for opening the issue! Could you provide a reproduction on stackblitz or provide a link to a public repro?

**Reply by jy95** [reply]
@valentinpalkovic My repo is public : https://github.com/jy95/ngx-json-schema-viewer 
So stackblitz link is https://stackblitz.com/github/jy95/ngx-json-schema-viewer

Run following commands at the root level & you will get the error I have in console : 
```bash
// Install dependancies
npm install
// Build the lib so that storybook can use it
npm run buildLib
// Launch Storybook
npm run storybook
```

Chromatic is also set in this repo ;)
( https://github.com/jy95/ngx-json-schema-viewer/blob/master/.github/workflows/chromatic.yml )
